tests/pfcwd_test.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+from click.testing import CliRunner
+from utilities_common.db import Db
+from pfcwd_input.pfcwd_test_vectors import testData
```

Comment:
Add whitespace to separate the imports per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:

> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

And group alphabetically in each group (these are already in alphabetical order). This should read:

```python
import os
import sys

from click.testing import CliRunner
from utilities_common.db import Db

from pfcwd_input.pfcwd_test_vectors import testData
```